src/platform/windows/display_vram.cpp

```diff
@@ -1239,6 +1246,10 @@ namespace platf::dxgi {
       old_surface_delayed_destruction.reset();
     }
 
+    if (img_out) {
+      img_out->frame_timestamp = frame_timestamp;
```

Comment:
Can you also plumb this into `display_ram_t`?

Reply:
Yes, if you're ok with the overall structure.

Reply:
Yeah, the approach looks good to me.

Reply:
Done.